HealthcareSystem/PatientWebApp/wwwroot/html/index.html

```diff
@@ -15,19 +15,50 @@
     <title>Welcome</title>
 </head>
 <body>
-    <div style="width: 50%; height:100%; float:left; position:fixed">
+
+    <div style="width: 100%; height:100%;">
+
         <div id="carousel" class="carousel slide" data-ride="carousel">
             <div class="carousel-inner">
+
+                <div style="width:800px; left:23%; top:60%; position:absolute" class="carousel-caption text-primary ">
+                    <h1 class="display-4" style="font-size:40px">Welcome! We care about your health!</h1>
+                    <br />
+                    <br />
+                    <table style="margin-left:130px">
+                        <tr>
+                            <td>
+                                <p style="font-size:22px; text-align:right; margin-bottom:30px; margin-top:15px">Need an account?</p>
+                            </td>
+                            <td>
+                                <input style="width:160px; height:50px" margin-bottom:30px" class="btn-light text-primary" type="button" onclick="location.href='/html/patient_registration.html';" value="Register" />
+                            </td>
```

Comment:
The buttons also need the btn class from bootstrap as well (that one shapes them and gives them that floaty effect, the btn-light just colours them).